openlr/openlr_match_quality/openlr_assessment_tool/main.cpp

```diff
@@ -6,11 +6,30 @@
 
 #include <QApplication>
 
+namespace
+{
 DEFINE_string(resources_path, "", "Path to resources directory");
 DEFINE_string(data_path, "", "Path to data directory");
+DEFINE_string(login, "", "Login string");
+DEFINE_string(paswd, "", "Password string");
+DEFINE_string(url, "", "Url to a partner map");
+
+bool ValidateStringFlag(char const * flagName, std::string const & val)
+{
+  if (!val.empty())
+    return true;
+
+  LOG(LERROR, (flagName, "cannot be empty. Please specify a proper", flagName));
```

Comment:
fprintf to stderr, please. Note! Logging system may not be initialized at this point, fprintf is much safer.

Reply:
Done.